fhir-openapi/src/main/webapp/META-INF/openapi.json

```diff
@@ -3,7 +3,7 @@
     "openapi": "3.0.0",
     "info": {
         "title": "Simplified FHIR API",
-        "description": "A simplified version of the FHIR API",
+        "description": "A simplified version of the HL7 FHIR API",
         "version": "4.0.0"
```

Comment:
Should the title match our IBM FHIR Server? or update to 4.0.1 to match the spec?

Reply:
IMHO this is mostly independent of our implementation...we don't have our own "flavor" of FHIR.  However, due to complexities with representing FHIR in Swagger/OpenAPI, this is not a representation of the *full* FHIR API.  This is why I settled on "A simplified version of the HL7 FHIR API".

However, I do agree that we should update it to version 4.0.0.

Also, I've been meaning to add a README for this module so that others can have some clue about what it does...I'll try adding that tonight and so please hold off on merging this one for now.